tests/dragonfly/instance.py

```diff
@@ -143,13 +144,6 @@ def _wait_for_server(self):
 
         self.log_files = self.get_logs_from_psutil()
```

Comment:
we can remove this I think -- this is not used anymore -- plz double check

Reply:
it is used in tests

Reply:
yes that's true in replication. So why do yet introduce another way to get the logs here and rely on the same mechanism? What's the value of adding this?

Reply:
@adiholden what's your take? Do we need (yet) another way to retrieve log files ?

Reply:
@kostasrim I dont understand what is your suggestion. How can we we know what are the log files of specific dragonfly instance? 
The change Borys introduced just creates a directory for a unique each test, but all the instances logs goes to this dir, also if we stop and start the instance.

Reply:
@adiholden When a test fails, we print the log names for that test. We do this for each separate test that fails. Moreover, we still need to look on the CI because we need to see what the failure was (did it fail on an assertion ? did it fail somehwere else ?) so my point is that we always end up looking on the CI one way or another. Ignore me for now :)